bundle/manifests/cryostat-operator.clusterserviceversion.yaml

```diff
@@ -576,10 +576,13 @@ spec:
       - description: Options to configure the Cryostat application's database.
         displayName: Database Options
         path: databaseOptions
-      - description: Name of the secret containing database keys. This secret must
+      - description: 'Name of the secret containing database keys. This secret must
           contain a CONNECTION_KEY secret which is the database connection password,
           and an ENCRYPTION_KEY secret which is the key used to encrypt sensitive
           data stored within the database, such as the target credentials keyring.
+          This field cannot be updated. It is recommended that the secret should be
+          marked as immutable to avoid accidental changes to secret''s data. More
```

Comment:
Something weird happening here with `''`

Reply:
I guess that single quote `'` is used to escape the `'s` in YAML? If I do, `make install && kubectl explain cryostat.spec.databaseOptions`, it looks alright.

```
GROUP:      operator.cryostat.io
KIND:       Cryostat
VERSION:    v1beta2

FIELD: databaseOptions <Object>

DESCRIPTION:
    Options to configure the Cryostat application's database.
    
FIELDS:
  secretName	<string>
    Name of the secret containing database keys. This secret must contain a
    CONNECTION_KEY secret which is the database connection password, and an
    ENCRYPTION_KEY secret which is the key used to encrypt sensitive data stored
    within the database, such as the target credentials keyring. This field
    cannot be updated. It is recommended that the secret should be marked as
    immutable to avoid accidental changes to secret's data. More details:
    https://kubernetes.io/docs/concepts/configuration/secret/#secret-immutable
```